app/src/main/java/com/example/music_app/constants/AuthUrl.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.music_app.constants
+
+const val SPOTIFY_CLIENT_ID = "67f6367abcd3497db54043591d9e7c9f"
+const val CODE = "code"
+const val SCOPE = "user-read-private user-read-email"
+const val REDIRECT_URL = "http://localhost:3000"
+
+const val STATE = "state"
+const val ERROR = "error"
```

Comment:
`Suggestion` I would recommend placing this constants directly inside of the file with `AppLoginWebViewClient ` and make them private

Reply:
The point of placing these constants in a common file for url is that they may be needed elsewhere in the program. They are connected with the url so I placed them in the AuthUrl file